prestopantry_app/views/my_pantry_page_view.py

```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from prestopantry_app.models.user_ingredients import UserIngredient
+
+@login_required(login_url='login')
+def display_pantry(request):
+    ingredients = UserIngredient.objects.all().filter(user=request.user)
```

Comment:
I'd use `.get(user=request.user)` instead of querying all objects and filtering on it

Reply:
`.get()`  can only pulls 1. if you try to pull more it errors out